### ISSUE TITLE
pressing Shift while highlighting text in console history causes spurious focus rectangles to show

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#6518]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.3.921
    OS Version      : macOS 10.15.3
    R Version       : 3.6.1

### Steps to reproduce the problem

Run a command so that there is some output showing in the console history. Then, try to highlight a line in the console history, and press the <kbd>Shift</kbd> key.

### Describe the problem in detail

Spurious focus rectangles are shown (including text which is not currently highlighted).

<img width="1140" alt="Screen Shot 2020-03-24 at 10 43 31 AM" src="https://user-images.githubusercontent.com/1976582/77459212-9936b500-6dbc-11ea-808d-35aa2bc0ab09.png">

### Describe the behavior you expected

Since text is currently highlighted, focus rectangles should be unnecessary. Or, at least, the display / presentation of focus rectangles here could be adjusted.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
See also #7242

**Comment by gtritchie** [comment]
Note, if we ever implement a way to select console output entirely via keyboard, might have to revisit focus rectangle behavior for the console; see #6077

**Comment by R-Hannibal** [comment]
Verified Fixed on: RStudio Version - 1.4.724, R Version - 4.0.2, Windows 10 x64 - Desktop

Focus rectangles are no longer displayed when pressing the Shift key with text highlighted in the console, tested with the 'Always show focus outlines' toggled both off and on..
